osquery/management.py

```diff
@@ -57,6 +68,18 @@ def __init__(self, path=None):
 
         # Disable logging for the thrift module (can be loud).
         logging.getLogger('thrift').addHandler(logging.NullHandler())
+        if sys.platform == "win32":
+            # Windows fails to spawn if the pidfile already exists
+            self._pidfile = (None, tempfile.gettempdir() + '\\pyosqpid-' +
+                             str(random.randint(10000, 20000)))
+            pipeName = r'\\.\pipe\pyosqsock-' + str(
+                random.randint(10000, 20000))
+            self._socket = (None, pipeName)
+        else:
+            self._socket = tempfile.mkstemp(prefix="pyosqsock")
+            self._pidfile = tempfile.mkstemp(prefix="pyosqpid")
+            with open(self._pidfile[1], "w") as fh:
+                fh.write("100000")
```

Comment:
I think you forgot to actually get the PID. Also maybe we should handle errors more gracefully here than allowing `IOError` to propagate.

Reply:
I think this was testing logic that I forgot to remove, going to just nuke this component, as I think on posix we don't need to worry about the pidfile.